Client/src/Common/Properties.cs

```diff
@@ -285,4 +324,19 @@ public string ConnectionString
   ///   The target name of the endpoint when ssl validation is disabled. Automatic if not set.
   /// </summary>
   public string TargetNameOverride { get; set; } = "";
+
+  /// <summary>
+  ///   Initial backoff from retries
+  /// </summary>
+  public TimeSpan RetryInitialBackoff { get; set; } = TimeSpan.FromSeconds(1);
+
+  /// <summary>
+  ///   Backoff multiplier for retries
+  /// </summary>
+  public double RetryBackoffMultiplier { get; set; } = 2;
+
+  /// <summary>
+  ///   Max backoff for retries
+  /// </summary>
+  public TimeSpan RetryMaxBackoff { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
only get ?

Reply:
Like any other field, it can be set from anywhere if you like.

Reply:
I don't think it is a good idea

Reply:
Why would they be different from the others ?

Reply:
because we need to do the rigth things at some point